numpy/add_newdocs.py

```diff
@@ -7756,3 +7758,36 @@ def luf(lamdaexpr, *args, **kwargs):
 
 add_newdoc('numpy.core.numerictypes', 'object_',
     """Any Python object.  Character code: 'O'.""")
+
```

Comment:
The ufunc ones end up in the generated files, there is also a special _add_newdoc_ufunc that can replace the doc of a ufunc.
Actually this should not be in the PR, I did it first in python but then realized its just as easily done in the C code for this case.